cinema_service/settings.py

```diff
@@ -86,12 +91,15 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.getenv("POSTGRES_DB", default="default"),
```

Comment:
Code quality issue: It's good that you're using environment variables for your database configuration. However, you should also provide a default value for all of them (like you did for `POSTGRES_DB`), not just for the database name. This way, your application can fall back to these default values if the environment variables are not set.